docs/modules/ROOT/pages/tmail-backend/jmap-extensions/calendarEventParsing.adoc

```diff
@@ -15,26 +15,47 @@ The associated object is empty.
 
 == CalendarEvent/parse
 
-This method bases on:
+This method is based on:
 - Parsing a Received MDN method as described in the RFC-9007
-- JMAP CalendarEvent RFC-8620
+- Draft JMAP CalendarEvent (https://datatracker.ietf.org/doc/html/draft-ietf-jmap-calendars)
 - iCalendar message RFC-5545
 
 The request object requires the following properties:
 
-- `accountId`: accountId of action user
-- `blobIds`: an array blobId of calendar which user wants to parse.
+- *accountId*: `Id` The id of the account to use.
+- *blobIds*: `[Id]` An array of calendar blob ids that the user wants to parse.
 
 The response object contains the following properties:
 
-- `accountId`: accountId of action user
-- `parsed`: a map of blobId and the corresponding calendar parse result.
-- `notFound`: an array of blobId is not found.
-- `notParsable`: an array of blobId has been found but is not parsable.
+- *accountId*: `Id` The id of the account to use.
+- *parsed*: a map of blob id and the corresponding calendar parse result.
+- *notFound*: an array of blob id is not found.
+- *notParsable*: an array of blob id has been found but is not parsable.
 
+The `parsed` value contains the following properties (the properties below match with the properties defined in rfc5545):
+
+- `title`: title of the event
```

Comment:
We don't have anymore the link with elements from iCalendar? SUMMARY, etc...
I didn't say to remove them, I said that we need a way for the reader to understand better where it is coming from :)